docs/api-gateway-v2/08-01-exposesecure.md

```diff
@@ -0,0 +1,239 @@
+
+Follow the instructions in the tabs to deploy an instance of the HttpBin service or a sample lambda function, expose them, and secure them with Oauth2 scopes.
+
+<div tabs>
+
+  <details>
+  <summary>
+  HttpBin - secure endpoints of a service
+  </summary>
+
+1. Deploy an instance of the HttpBin service:
+
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml
+  ```
+
+2. Expose the service and secure it by creating an APIRule CR:
+
+  ```
+  cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: httpbin
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    service:
+      name: httpbin-proxy
+      port: 8000
+      host: httpbin-proxy.kyma.local
+    rules:
+      - path: /.*
+        methods: ["GET"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["read"]
+      - path: /post
+        methods: ["POST"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["write"]
+  EOF
+  ```
+
+>**NOTE:** If you are running Kyma on Minikube, add `httpbin-proxy.kyma.local` to the entry with Minikube IP in your system's `/etc/hosts` file.
+
+The exposed service requires tokens with "read" scope for `GET` requests in the entire service and tokens with "write" scope for `POST` requests to the `/post` endpoint of the service.
+
+  </details>
+
+  <details>
+  <summary>
+  Secure a lambda function
+  </summary>
+
+1. Create a lambda function using the [supplied code](./assets/lambda.yaml):
+
+  ```
+  kubectl apply -f ./assets/lambda.yaml
+  ```
+
+2. Expose the lambda function and secure it by creating an APIRule CR:
+
+  ```
+  cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: lambda
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    service:
+      name: labmda-proxy
+      port: 8080
+      host: labmda-proxy.kyma.local
+    rules:
+      - path: /lambda
+        methods: ["GET"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["read"]
+  EOF
+  ```
+
+>**NOTE:** If you are running Kyma on Minikube, add `lambda-proxy.kyma.local` to the entry with Minikube IP in your system's `/etc/hosts` file.  
+
+The exposed lambda function requires all `GET` requests to have a valid token with the "read" scope.
+
+  </details>
+</div>
+
+## Access the secured resources
+
+Follow the instructions in the tabs to call the secured service or lambda functions using the tokens issued for the client you registered.
+
+<div tabs>
+
+  <details>
+  <summary>
+  Call secured endpoints of a service
+  </summary>
+
+1. Send a `GET` request with a token with the "read" scope to the HttpBin service:
```

Comment:
Is there a way to rephrase it to avoid so many `with`'s? (Also, lines 215 and 230.)